masim.h

```diff
@@ -1,9 +1,11 @@
 #ifndef _MASIM_H
 #define _MASIM_H
 
+#include <stdint.h>
+
 struct mregion {
 	char name[256];
-	size_t sz;
+	uint64_t sz;
```

Comment:
Again, isn't size of `size_t` on 64 bit machines are 8 bytes?  Why we need this change?